docs/env_setup/local/miniconda_conda_local_setup.md

```diff
@@ -9,16 +9,17 @@
 	- Ensure you add anaconda as a path environment variable (even if it says this option is not recommended)
 	- Ensure you do *not* register anaconda as the default version of Python.
 	- _Note_: These installation settings can always be changed posthoc.
-
-2) Create conda environment and install the code dependencies from the `env.yml` file:
+2. Create conda environment and install the code dependencies from the `env.yml` file:
 ```
 conda update conda
 conda init
-conda env create --file env.yml
+conda env create --file env_config/env.yml
 ```
-
-3) Using the virtual environment:
-
-`conda activate aeon`: activates the virtual environment; any commands now run within this terminal will take place within the virtual environment.
-
-`conda deactivate aeon`: deactivates the virtual environment.
+3. Optionally install development dependencies:
+```
+conda activate aeon
+conda env update --file env_config/env_dev.yml
+```
+4. Using the virtual environment:
+	- `conda activate aeon`: activates the virtual environment; any commands now run within this terminal will take place within the virtual environment.
```

Comment:
Large indentation?

Reply:
I think the markdown renders nicer this way - makes it look like a sublist belonging to '4.' than if it was non-indented.